keras_nlp/models/gemma/gemma_attention.py

```diff
@@ -87,28 +88,20 @@ def build(self, inputs_shape):
             (None, None, self.num_query_heads, self.head_dim)
         )
         self.softmax = keras.layers.Softmax(dtype="float32")
+
+        self.rope_layer = RotaryEmbedding(
+            max_wavelength=10000.0, dtype=self.dtype_policy
```

Comment:
nit, but let's start using the new `10_000.0` for large numbers, it's quite readable.

Reply:
Sounds good, I love `_` to separate large numbers!